weld/common/src/main/java/org/jboss/as/weld/interceptors/Jsr299BindingsInterceptor.java

```diff
@@ -41,10 +41,10 @@
 import org.jboss.weld.ejb.spi.InterceptorBindings;
 
 /**
- * Interceptor for applying the JSR-299 specific interceptor bindings.
+ * Jakarta Interceptors for applying Interceptors bindings.
  * <p/>
- * It is a separate interceptor, as it needs to be applied after all
- * the other existing interceptors.
+ * It is a separate Jakarta Interceptors, as it needs to be applied after all
+ * the other existing Jakarta Interceptors.
```

Comment:
Leave this as they were before:

It is a separate interceptor, as it needs to be applied after all the other existing interceptors.